themes/courses.labster.com/lms/templates/emails/activation_email.txt

```diff
@@ -14,7 +14,8 @@ https://${ site }/activate/${ key }
 http://${ site }/activate/${ key }
 % endif
 
-${_("If you didn't request this, you don't need to do anything; you won't "
-"receive any more email from us. Please do not reply to this e-mail; "
-"if you require assistance, check the help section of the "
-"{platform_name} website.").format(platform_name=settings.PLATFORM_NAME)}
+${_("If you have any questions please contact: {support_email}").format(
+    support_email=settings.BUGS_EMAIL
```

Comment:
@sidorovdmitry could you make sure `BUGS_EMAIL` contains `support@labster.com` email